README.md

```diff
@@ -3,6 +3,42 @@
 
 A library for calling C functions from Go without Cgo.
 
+## Example
+
+```go
+package main
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/ebitengine/purego"
+)
+
+func getSystemLibrary() string {
+	switch runtime.GOOS {
+	case "darwin":
+		return "/usr/lib/libSystem.B.dylib"
+	case "linux":
+		return "libc.so.6"
```

Comment:
Do we need `.6`?

Reply:
On Ubuntu where I tested, yes

Reply:
Also [this](https://stackoverflow.com/questions/6841311/using-dlopen-to-get-handle-of-libc-memory-allocation-functions) StackOverflow issue says libc.so is a linker script not an elf file.

Reply:
You mean `dlopen("libc.so")` without `.6` did not work?

Reply:
I remembered that there is an environment that such a symbolic link doesn't exist. NVM

Reply:
Without the .6 it fails on Ubuntu